.github/workflows/visual-tests.yaml

```diff
@@ -5,7 +5,7 @@ on:
     types: [ opened, synchronize ]
 
 env:
-  TEST_PATHS: "/ /career/yuya-yoshisue /brands/consumer-health-beauty /healthy-thinking/6-things-you-didnt-know-about-your-oral-health /brands /about/history /about/business-performance"
+  TEST_PATHS: "/*"
```

Comment:
I am not sure how much it will take to run this for each and every page

Reply:
Don't worry, it seems this does not work unfortunately